pages/_app.js

```diff
@@ -29,6 +33,8 @@ export default function App({ Component, pageProps }) {
     }
   );
 
+  const [demoMode, setDemoMode] = useState(false);
```

Comment:
This is totally petty (i know 😂), but I probably would have added this useState to the other useStates above as well (above the useLocalStorageState). But of course you can also leave it as it is.

Reply:
your suggestion is prettier😄